cirq-core/cirq/optimizers/merge_interactions_to_sqrt_iswap_test.py

```diff
@@ -0,0 +1,187 @@
+from typing import Callable, List
+
+import cirq
+
+
+def assert_optimizes(before: cirq.Circuit, expected: cirq.Circuit, **kwargs):
+    actual = cirq.Circuit(before)
```

Comment:
```suggestion
    actual = before.copy()
```

Reply:
Done.  This was copied from old test code.